src/lib.rs

```diff
@@ -53,7 +53,7 @@
 //! # }
```

Comment:
While I'm OK with `.unwrap()` for the hidden (`# `-prefixed) doc comments, we might want to *not* encourage unmanaged panics in visible demo lines?

Reply:
OK, fair enough!